pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/UploadStatusLoader.kt

```diff
@@ -62,53 +63,66 @@ class UploadStatusLoader: CosmosLoader() {
             sqlQuery.append(" and r.content.filename = '$fileName'")
         }
 
+        status?.run {
+            sqlQuery.append(" and r.stageInfo.status = '$status'")
+        }
+
         dateStart?.run {
             val dateStartEpochSecs = DateUtils.getEpochFromDateString(dateStart, "date_start")
-            sqlQuery.append(" and r._ts >= $dateStartEpochSecs")
+            sqlQuery.append(" and r.dexIngestDateTime >= $dateStartEpochSecs")
         }
         dateEnd?.run {
             val dateEndEpochSecs = DateUtils.getEpochFromDateString(dateEnd, "date_end")
-            sqlQuery.append(" and r._ts < $dateEndEpochSecs")
+            sqlQuery.append(" and r.dexIngestDateTime < $dateEndEpochSecs")
         }
 
         sqlQuery.append(" group by r.uploadId")
 
         // Check the sort field as well to add them to the group by clause
         sortBy.run {
             val sortField = when (sortBy) {
-                //"date" -> "_ts" // "group  by _ts" is already added by default above
+                "date" -> "dexIngestDateTime" // "group  by _ts" is already added by default above
                 "fileName" -> "content.filename"
-                "dataStreamId" -> "destinationId"
-                "dataStreamRoute" -> "eventType"
-                "stageName" -> "stageName"
-                "jurisdiction" -> "content.metadata.meta_ext_source"
+                "dataStreamId" -> "dataStreamId"
+                "dataStreamRoute" -> "dataStreamRoute"
+                "stageName" -> "stageInfo.service"
```

Comment:
Please rename "stageName" to "service" -> "stageInfo.service" and add "action" -> "stageInfo.action".

Reply:
Updated according to the suggestions